prov/verbs/src/fi_verbs.c

```diff
@@ -366,5 +366,9 @@ static void fi_ibv_fini(void)
 
 VERBS_INI
 {
+	fi_param_define(&fi_ibv_prov, "iface", FI_PARAM_STRING,
+			"prefix or full name of network interface associated "
+			"with IB device (default: ib)");
+
```

Comment:
Firstly, yes. Secondary, connection manager of ep_rdm should be bound to valid sockaddr_in for listening, it is resolved via interface associated with IB device